README.adoc

```diff
@@ -37,7 +37,7 @@ coff:ee can also be found in the official maven repository as well, it is enough
 <dependencyManagement>
     <dependency>
         <groupId>hu.icellmobilsoft.coffee</groupId>
-        <artifactId>coffee-bom</artifactId>
+        <artifactId>coffee-bom-all</artifactId>
```

Comment:
igy ha mar kulon van valasztva a sajat muduljai akkor szerintem a coffee-bom-project kellene ide. Tehat a cel hogy a projekt aki hasznalja a coffee-t annak csak a coffee moduljait kelljen behuzni, a `coffee-bom-all` a coffee-n beluli hasznalatra kell

Reply:
Igen, ki is próbáltam, így van, átírom.